### PR TITLE
Pimcore x docs

### LINKED ISSUE #7195: [DOCS] for 7.0
- removing PHP templates
- removed features
...
...

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #7195

### CODE REVIEW

File: doc/Development_Documentation/03_Documents/01_Editables/06_Block.md [hunk 1]
```diff
@@ -119,19 +119,14 @@ If you want to wrap buttons in a div or change the Position.
...
- 
- 
-var_dump("Bsp #1: " . $image->getSrc());
+```twig
+
+{# load document #}
+{% set document = pimcore_document_by_path('/en/More-Stuff/Developers-Corner/Galleries') %}
+
+{# get the first picture from the first "gallery-carousel" brick #}
+{% set image = document.getEditable('content').getElement('gallery-single-images')[5].getBlock('gallery').getElements()[0].getImage('image') %}
```
Comment (jdreesen) [inline doc/Development_Documentation/03_Documents/01_Editables/06_Block.md:128]: The comment and the previous code indicate that it should be like this:

```suggestion
{% set image = document.getEditable('content').getElement('gallery-single-images')[0].getBlock('gallery').getElements()[0].getImage('image') %}
```

File: doc/Development_Documentation/04_Assets/03_Working_with_Thumbnails/03_Video_Thumbnails.md [hunk 2]
```diff
@@ -51,11 +51,11 @@ As soon as you define transformations based on the bitrates in thumbnail config,
 The `.mpd` file will be referenced in  generated `<video>` Tag.
 
 However, you have to include a polyfill for all major browsers to support Adaptive  bitrate video-streaming: https://github.com/Dash-Industry-Forum/dash.js
-```php
+```twig
 {{ pimcore_video('campaignVideo', {
         width: auto,
         height: auto,
-        thumbnail: 'new',
```
Comment (jdreesen) [inline doc/Development_Documentation/04_Assets/03_Working_with_Thumbnails/03_Video_Thumbnails.md:58]: trailing commas are totally fine in twig ;)

File: doc/Development_Documentation/03_Documents/01_Editables/06_Block.md (hunk 1 above)
Comment (dvesh3) [reply doc/Development_Documentation/03_Documents/01_Editables/06_Block.md:128]: No...on new demo this element is on 5th index

Comment (brusch) [reply doc/Development_Documentation/03_Documents/01_Editables/06_Block.md:128]: Just reverted 😊

File: doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/README.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ You can use this functionality for your own application, and also to control the
 careful!).
 
 If you don't need the transactional tagging functionality as used in the core you're free to use a custom cache system as
-[provided by Symfony](http://symfony.com/blog/new-in-symfony-3-1-cache-component) but be aware that custom caches are not 
+[provided by Symfony](https://symfony.com/blog/new-in-symfony-3-1-cache-component) but be aware that custom caches are not
```
Comment (dvesh3) [inline doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/README.md]: please change the link to https://symfony.com/doc/5.2/components/cache.html

Comment (brusch) [reply doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/README.md]: Check 👍😊

**Pull request merged**